docs/src/main/sphinx/release/release-363.md

```diff
@@ -0,0 +1,63 @@
+# Release 363 (XX Oct 2021)
+
+## General
+
+* Reduce query memory usage when spill occurs. ({issue}`9270`)
+* Reduce join memory usage. ({issue}`9327`)
+* Reduce query memory usage during join unspilling. ({issue}`9275`)
+* Fix potential data loss in query results when clients retry requests to the coordinator. ({issue}`9453`)
+* Fix decimal arithmetic to not produce negative zeros. Previously comparing two zero decimal
+  values might have returned `false` because of dual zero representation. ({issue}`8985`)
+* Fix `SHOW ROLES` failure when there are no roles to display. ({issue}`9319`)
+* Fix `EXPLAIN ANALYZE` to show estimates. ({issue}`9396`)
+
+## Python client
+
+* Fix column type reported in `cursor.description` for `time with time zone` column. ({issue}`9460`)
+
+## BigQuery connector
+
+* Fix failure for queries where predicate on `geography`, `array` or `struct` column is pushed down to BigQuery. ({issue}`9391`)
+
+## Cassandra connector
+
+* Add support for Cassandra TUPLE type. ({issue}`8570`)
+
+## Elasticsearch connector
+
+* Add support for `scaled_float` type. ({issue}`9358`)
+
+## Hive connector
+
+* Improve query performance when reading Parquet data with predicate on a `decimal` column. ({issue}`9338`)
+* Fix insert overwrite operation on S3 backing storage. ({issue}`9234`)
+* Fix `Failed reading parquet data: Socket is closed by peer` query failure when reading from Parquet table with a predicate. ({issue}`9097`)
+* Fix query failure when updating or deleting from ORC ACID transactional table that has some rows deleted since last major compaction. ({issue}`9354`)
+* Fix failure when reading large Parquet files. ({issue}`9469`)
+
+## Iceberg connector
+
+* Add support for comment when adding a new column. ({issue}`9123`)
```

Comment:
This feels like "Fix", since previously the comment was silently ignored.

Reply:
How about:

> Fix support for comments when adding a new column. Previously, they were silently ignored.